libs/ledger-live-common/src/DataModel.ts

```diff
@@ -40,20 +40,24 @@ export function createDataModel<R, M>(schema: DataSchema<R, M>): DataModel<R, M>
   function decodeModel(raw) {
     let { data } = raw;
 
-    if (data.currencyId == "crypto_org" && !data.cosmosResources) {
-      data.cosmosResources = {
-        delegations: [],
-        redelegations: [],
-        unbondings: [],
-        delegatedBalance: new BigNumber(0),
-        pendingRewardsBalance: new BigNumber(0),
-        unbondingBalance: new BigNumber(0),
-        withdrawAddress: data.freshAddress,
-        sequence: 0,
-      };
-    }
-    for (let i = raw.version; i < version; i++) {
-      data = migrations[i](data);
+    if (data.currencyId === "aptos" && migrations.length) {
+      data = migrations[0](data);
+    } else {
```

Comment:
@hedi-edelbloute The previous code NEVER ran the migrations because `version === migrations.length === 1` so it never entered the `for` loop.
I believe this is legacy code that is no longer being used but this was the way I found to introduce a migration for Aptos without breaking what was already there.
Let me know your opinion.